source/OptimaJet.DWKit.StarterApplication/Services/Workflow/WorkflowProductActionProvider.cs

```diff
@@ -313,6 +315,39 @@ private async Task UpdateProductTransitionAsync(ProcessInstance processInstance,
             }
         }
 
+        private async Task GooglePlay_UpdatPublishLinkAsync(ProcessInstance processInstance, WorkflowRuntime runtime, string actionParameter, CancellationToken token)
```

Comment:
do you have a pattern for testing these workflow actions?
would be good to get these tested at some point

Reply:
Yeah, I was thinking about that this morning.  They are executed by Workflow.  I might need to move some of this to a utility class so that it can more easily be tested.

Reply:
I mentioned during standup, I am going to migrate some of the work on 30929 to WorkflowAction and I will refactor to allow for testing.